src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPage.epics.js

```diff
@@ -122,3 +124,36 @@ export const openEnrollmentPageEpic = (action$: InputObservable, store: ReduxSto
         },
         ),
     );
+
+export const clickLinkedRecordEpic = (action$: InputObservable, store: ReduxStore, { history }: ApiUtils) =>
+    action$.pipe(
+        ofType(enrollmentPageActionTypes.LINKED_RECORD_CLICK),
+        flatMap(({ payload }) => {
+            let url;
+            const { programId, orgUnitId } = payload;
+            if (payload.eventId) {
+                const recordProgram = getProgramFromProgramIdThrowIfNotFound(programId);
+                if (recordProgram instanceof EventProgram) {
+                    url = `/viewEvent?viewEventId=${payload.eventId}`;
+                } else {
+                    url = `/enrollmentEventEdit?${buildUrlQueryString({
+                        orgUnitId,
+                        programId,
```

Comment:
Hey, 
The URL parameters were simplified on the `enrollmentEventEdit` page and merged in [DHIS2-13253](https://github.com/dhis2/capture-app/pull/2845). The `programId` can be now safely removed. 
Thank you!